docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,72 @@
+# Why Toolpad?
+
+<p class="description">Explains our motivation for building Toolpad, where it differs from other tools and how it could be useful to you.</p>
```

Comment:
Iterating on the suggestion above to: 1) center the needs of the reader (they probably don't really care about "our motivation") and 2) specify what kind of tools we're talking about

```suggestion
<p class="description">Learn how Toolpad differs from the other internal tool builders, and how it can be useful to you.</p>
```